### PR TITLE
Validate visitor el support

### LINKED ISSUE #14232: Update JSP core ELSupport method calls and remove overlays in EL 4.0 implementation
Changes in open liberty proposed here: https://github.com/OpenLiberty/open-liberty/issues/14111

https://bz.apache.org/bugzilla/show_bug.cgi?id=64778

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #14232

This to me looks to be the cleanest solution to help us with forwarding development.

1) Update ValidateVisitor to call into the new EL methods that exist in ExpressionLanguage 3.0+.  Just pass in `null` for the `ELContext` as that ensure we get the same behavior as we have previously and is consistent across implementations.
2) Add these methods into the Expression Lanague EL 2.2 implementation and just call through to the existing `coerceX` methods. This means there is no behavior change in EL 2.2, the same method ends up getting called n the end. We can't port the methods from Expression Language 3.0+ to Expression Language 2.2 since the functionality they use is specific to Expression Language 3.0+ `convertToType`.
3) Update the bnd.bnd for ExpressionLanauge 2.2 and ExpressionLanauge  4.0 to disable trace instrumentation `instrument.ffdc: false` because having this means we're only instrumentation the classes we have in overlays which doesn't make a whole lot of sense.
4) Remove the ELSupport overlay in the Expression Language 4.0 implementation which had added the old methods called from ValidateVisitor as they did not exist any longer in the implementation.

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Review by isaacrivriv — APPROVED** [review]
Looks good to me. Thanks!

**Comment by epj** [comment]
I suggest running more buckets with this, all the tests for EL 2.2 and 4, JSP tests for 2.2 and 2.3.

**Review by volosied — APPROVED** [review]
I agree with the others. This looks like a the best way of handing this issue, and the changes itself looks good. 

I don't have any comments otherwise.

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsp_fat,com.ibm.ws.jsp_bvt,com.ibm.ws.jsp_fat_jsp.2.3.toleration,com.ibm.ws.jsp_fat_lWAS,com.ibm.ws.jsp.2.3_fat,com.ibm.ws.el.3.0_fat

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsp_fat,com.ibm.ws.jsp_fat_jsp.2.3.toleration,com.ibm.ws.jsp_fat_lWAS,com.ibm.ws.jsp.2.3_fat,com.ibm.ws.el.3.0_fat

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsp_fat,com.ibm.ws.jsp_fat_jsp.2.3.toleration,com.ibm.ws.jsp_fat_lWAS,com.ibm.ws.jsp.2.3_fat,com.ibm.ws.el.3.0_fat

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsp_fat,com.ibm.ws.jsp_fat_jsp.2.3.toleration,com.ibm.ws.jsp_fat_lWAS,com.ibm.ws.jsp.2.3_fat,com.ibm.ws.el.3.0_fat

**Comment by pnicolucci** [comment]
#run-libby-bot

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsp_fat,com.ibm.ws.jsp_fat_jsp.2.3.toleration,com.ibm.ws.jsp_fat_lWAS,com.ibm.ws.jsp.2.3_fat,com.ibm.ws.el.3.0_fat

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsp_fat,com.ibm.ws.jsp_fat_jsp.2.3.toleration,com.ibm.ws.jsp_fat_lWAS,com.ibm.ws.jsp.2.3_fat,com.ibm.ws.el.3.0_fat

**Comment by pnicolucci** [comment]
#run-libby-bot

File: dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/translator/visitor/validator/ValidateVisitor.java [hunk 1]
```diff
@@ -1771,19 +1771,19 @@ else if (checkDeferred && ((ELNode.Root) node).getType() == '#') {
 	//method used to replace ELSupport.checkType
```
Comment (volosied) [inline dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/translator/visitor/validator/ValidateVisitor.java:1771]: Copyright

**Comment by volosied** [comment]
Code looks good! Thanks again

**Comment by pnicolucci** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.jsp_fat,com.ibm.ws.jsp_fat_jsp.2.3.toleration,com.ibm.ws.jsp_fat_lWAS,com.ibm.ws.jsp.2.3_fat,com.ibm.ws.el.3.0_fat

**Comment by pnicolucci** [comment]
#run-libby-bot

**Pull request merged**